conda_gitenv/deploy.py

```diff
@@ -54,6 +54,11 @@ def deploy_tag(repo, tag_name, target):
 
 
 def create_env(pkgs, target, pkg_cache):
+    # Empty package cache so that it will reinitialised.
+    conda.install.package_cache_ = {}
```

Comment:
https://github.com/conda/conda/blob/4.1.12/conda/install.py#L714
Yep, but if `package_cache_` is already set it will just return that one which will cause problems if your package_cache is already pointing to `$PREFIX/pkgs` or pointing to a different `< target >/.package_cache`